src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "Troubleshooting Guide"
```

Comment:
Nit: for consistency, let's also change "Quickstart guide" to "Quickstart Guide" and "Online resources" to "Online Resources". I will submit a PR for that.